### ISSUE TITLE
A11y_AzureDataStudioAAE_PG1_Compute + Storage_Keyboard: "Save" & "Discard" button are not getting the instant keyboard focus after updating the edit fields.

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15759]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG2.4.3;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio - AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "PG1" and open manage and tab to "Compute + Storage" page
5. Edit any Spin button and tab to save button

### Actual:
"Save" & "Discard" button are not getting the instant keyboard focus after updating the edit fields.

### Expected:
"Save" & "Discard" button should get the instant keyboard focus after updating the edit fields.

### User Impact:
Keyboard users will get confused if the button do not focus in logical sequenced order.

https://user-images.githubusercontent.com/85863543/127598602-26f4264d-541c-4640-b057-b7d14f05582a.mp4

**Comment by nasc17** [comment]
Would like clarification on what should happen here.  
Should the focus go to the buttons right after the user makes a change to any one text box? Couldn't that steal focus away in the middle of typing something or changing values? What if the user wanted to continue down the fields to edit instead of saving right away?

**Comment by nasc17** [comment]
@krrish2021 Will be closing, please link to requirement for this issue and reopen.

**Comment by krrish2021** [comment]
@nasc17 "Save" & "Discard" button should be present at the bottom of the page and not on top of the page so that after editing all the fields, the focus should shift to "Save" & "Discard" button. Please reopen the bug.

**Comment by candiceye** [comment]
@krrish2021 Closing this for now, as this Save button is present in other areas in ADS and have not been flagged before; could you please provide a link to the documentation/WCAG requirements for this issue?

**Comment by alanrenmsft** [comment]
@krrish2021 the location of the button shouldn't matter, right? for example, Outook, Word and even notepad, the actions/menus are all at the top of the UI.
